home.html

```diff
@@ -0,0 +1,129 @@
+<!doctype html>
```

Comment:
This file should probably be renamed to index.html, as it will be the default page to open.